### PR TITLE
override `Api`

### PR DESCRIPTION
**Pull request by aidan-starke**
### Details
- override default symbol & decimals, `signedExtensions`, remove `UNIT` using `useApiOverride` hook
- [useApiOverride](https://github.com/cennznet/app-portal/blob/2b24d37bb1be2c1e9dfdab71a86e7674d76e5f93/src/libs/hooks/useApiOverride.ts)
- restore `storage` route (chain state)
- remove `UNIT` from `Metadata` table

### Notes
 - Need to update metadata (on settings page) in order to sign extrinsic with polkadot extension

### CODE REVIEW

File: polkadot/packages/react-api/src/Api.tsx [hunk 1]
```diff
@@ -131,8 +130,8 @@ async function loadOnReady (api: ApiPromise, endpoint: LinkOption | null, inject
   const ss58Format = settings.prefix === -1
     ? chainSS58
     : settings.prefix;
-  const tokenSymbol = properties.tokenSymbol.unwrapOr([formatBalance.getDefaults().unit, ...DEFAULT_AUX]);
-  const tokenDecimals = properties.tokenDecimals.unwrapOr([DEFAULT_DECIMALS]);
+  const tokenSymbol = registry.createType('Text', 'CPAY');
+  const tokenDecimals = [DEFAULT_DECIMALS];
```
Comment (ken-futureverse) [inline polkadot/packages/react-api/src/Api.tsx]: so `tokenSymbol` and `tokenDecimals` are all coming from `properties` prop, is there no way we can do it without changing the `react-api` package directly?

**Review by ken-futureverse — COMMENTED** [review]
nicely done, but do we still need `state, setState` in `Api.tsx`?

**Comment by aidan-starke** [comment]
> nicely done, but do we still need `state, setState` in `Api.tsx`?

this is to avoid having to edit `NodeName` which is a part of `react-query` package

**Review by ken-futureverse — APPROVED** [review]
Looks good, but I'd like to remove the patch in `packages/react-api` that expose `state` and `setState` for `useApi` hook. This can be in a separate PR

**Pull request merged**